packages/nextjs/pages/_app.tsx

```diff
@@ -10,7 +10,13 @@ import { appChains } from "~~/services/web3/wagmiConnectors";
 import { wagmiClient } from "~~/services/web3/wagmiClient";
 import Header from "~~/components/Header";
 
+import { useAppStore } from "~~/services/store/store";
+import { useEthPrice } from "~~/hooks/scaffold-eth";
+
 const ScaffoldEthApp = ({ Component, pageProps }: AppProps) => {
+  const price = useEthPrice();
+  useAppStore(state => state.priceSlice.setPriceData({ price: price }));
```

Comment:
I'm not sure if it's the best place to set it. Please let me know if there is a better place.

Reply:
Changed!